roles/edpm_ovn/tasks/configure.yml

```diff
@@ -82,26 +82,32 @@
 
 - name: Configure neutron configuration files
   block:
-    - name: Ensure {{ edpm_ovn_neutron_config_dir }}/etc/neutron/plugins/networking-ovn exists
-      ansible.builtin.file:
-        path: "{{ edpm_ovn_neutron_config_dir }}/etc/neutron/plugins/networking-ovn"
-        state: directory
-        recurse: true
-
-    - name: Render neutron.conf file
+    - name: render neutron config files
       ansible.builtin.template:
-        src: "neutron.conf.j2"
-        dest: "{{ edpm_ovn_neutron_config_dir }}/etc/neutron/neutron.conf"
-        mode: "0644"
+        src: "{{ item.src }}"
+        dest: "{{ edpm_ovn_neutron_metadata_agent_config_dir }}/{{ item.dest }}"
+        setype: "container_file_t"
+        mode: 0644
```

Comment:
Putting the permissions in quotes is encouraged. That way we can be sure that ansible handles the parsing.